language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/helpers/UserManagementServiceImpl.scala

```diff
@@ -47,6 +47,11 @@ final class UserManagementServiceImpl extends UserManagementService with FakeAut
 
   override def listUserRights(request: ListUserRightsRequest): Future[ListUserRightsResponse] =
     record(request)(ListUserRightsResponse.defaultInstance)
+
+  override def updateUser(request: UpdateUserRequest): Future[UpdateUserResponse] = {
+    // TODO: um-for-hub
+    ???
```

Comment:
These are mocks, so a trivial impl needs to be provided

Reply:
Ok